manifests/stage-multus-cni/0050-multus-ds.yml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
```

Comment:
anything changed here ? (github marked this file as "changed since last review")

Reply:
added nodeAffinity, to prevent deploying on master